### PR TITLE
DOC-2171: the TinyMCE 6.7 release notes plus 6.7-specific changes to the TinyMCE documentation

### PR DESCRIPTION
**Pull request by brian-forte**
DOC-2171: the TinyMCE 6.7 release notes plus 6.7-specific changes to the TinyMCE documentation

Changes:
* So many changes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] `modules/ROOT/nav.adoc` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/partials/configuration/tableofcontents_orderedlist_type.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+=== Example: using `tableofcontents_orderedlist` to render a Table of Contents as uppercase Roman numerals
+[source, js]
+----
+tinymce.init({
+  selector: "textarea", // change this value according to your HTML
+  plugins: "tableofcontents",
+  toolbar: "tableofcontents",
+  tableofcontents_orderedlist: true, // required to enable tableofcontents_orderedlist_type configuration.
+	tableofcontents_orderedlist_type: 'I',
+});
```
Comment (kemister85) [inline modules/ROOT/partials/configuration/tableofcontents_orderedlist_type.adoc:51]: Indentation looks incorrect for the last line?

```suggestion
tinymce.init({
  selector: "textarea", // change this value according to your HTML
  plugins: "tableofcontents",
  toolbar: "tableofcontents",
  tableofcontents_orderedlist: true, // required to enable tableofcontents_orderedlist_type configuration.
	tableofcontents_orderedlist_type: 'I',
});
```

File: modules/ROOT/partials/configuration/tableofcontents_orderedlist_type.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+----
```
Comment (kemister85) [inline modules/ROOT/partials/configuration/tableofcontents_orderedlist_type.adoc:52]:
```suggestion
----
tinymce.init({
  selector: "textarea", // change this value according to your HTML
  plugins: "tableofcontents",
  toolbar: "tableofcontents",
  tableofcontents_orderedlist: true, // required to enable tableofcontents_orderedlist_type configuration.
  tableofcontents_orderedlist_type: 'I',
});
----
```

Comment (TheSpyder) [reply modules/ROOT/partials/configuration/tableofcontents_orderedlist_type.adoc:52]: yes it's using a tab rather than spaces, good catch

**Review by TheSpyder — APPROVED** [review]

File: modules/ROOT/partials/configuration/tableofcontents_includeheader.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+=== Example: using `tableofcontents_includeheader` to turn the Table of Contents header string off
+[source, js]
+----
+tinymce.init({
+  selector: "textarea", // change this value according to your HTML
+  plugins: "tableofcontents",
+  toolbar: "tableofcontents",
+	tableofcontents_includeheader: false,
+});
+----
```
Comment (kemister85) [inline modules/ROOT/partials/configuration/tableofcontents_includeheader.adoc:23]:
```suggestion
----
tinymce.init({
  selector: "textarea", // change this value according to your HTML
  plugins: "tableofcontents",
  toolbar: "tableofcontents",
  tableofcontents_includeheader: false,
});
----
```

File: modules/ROOT/pages/6.7-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,941 @@
...
+
+[source, js]
+----
+tinymce.init({
+  selector: "textarea",
+  plugins: [ "help", ],
+  toolbar: "help",
+	help_accessibility: true, // default value is set to true.
+});
+----
```
Comment (kemister85) [inline modules/ROOT/pages/6.7-release-notes.adoc:536]:
```suggestion
[source, js]
----
tinymce.init({
  selector: "textarea",
  plugins: [ "help", ],
  toolbar: "help",
  help_accessibility: true, // default value is set to true.
});
----
```

**Review by kemister85 — APPROVED** [review]

**Review by tiny-ben-tran — APPROVED** [review]

**Pull request merged**